app/src/ui/check-runs/ci-check-run-list-item.tsx

```diff
@@ -67,25 +67,70 @@ export class CICheckRunListItem extends React.PureComponent<
     this.setState({ isMouseOverLogs: false })
   }
 
+  private isLoading = (): boolean => {
+    if (this.props.loadingActionLogs) {
+      return true
+    }
+
+    if (
+      this.props.loadingActionWorkflows &&
+      this.props.checkRun.actionsWorkflowRunId !== undefined
+    ) {
+      return true
+    }
+
+    return false
+  }
+
+  private renderCheckStatusSymbol = (): JSX.Element => {
+    const { checkRun, showLogs } = this.props
+
+    const stepLoader = (
+      <svg
+        fill="none"
+        height="16"
+        viewBox="0 0 16 16"
+        width="16"
+        className="spin"
+      >
+        <g strokeWidth="2">
+          <circle cx="8" cy="8" r="7" stroke="#959da5"></circle>
+          <path
+            d="m12.9497 3.05025c1.3128 1.31276 2.0503 3.09323 2.0503 4.94975 0 1.85651-.7375 3.637-2.0503 4.9497"
+            stroke="#fafbfc"
+          ></path>
+        </g>
+      </svg>
+    )
```

Comment:
It'd be nice to have this as a reusable component, but I guess we can wait until we want to actually reuse it

Reply:
Yeah. I would defiantly like to wait.. seeing as this really is only for waiting on logs.. and that may not be a thing anyways.